assets/themes/gruvbox/gruvbox.json

```diff
@@ -482,7 +482,7 @@
         "hidden": "#998b78ff",
         "hidden.background": "#4c4642ff",
         "hidden.border": "#544c48ff",
-        "hint": "#8c957dff",
+        "hint": "#6a695bff",
```

Comment:
Just making sure: is this change intentional?

Reply:
Yes, 100% :P